sw/host/opentitanlib/src/transport/cw310/usb.rs

```diff
@@ -298,9 +301,19 @@ impl Backend {
         Ok(status[0] & 0x01 != 0)
     }
 
-    // Erase the FPGA and prepare for programming.
-    fn fpga_erase(&self) -> Result<()> {
-        self.send_ctrl(Backend::CMD_FPGA_PROGRAM, Backend::PROGRAM_INIT, &[])?;
+    // Set the FPGA download speed and prepare for programming.
+    fn fpga_prepare(&self, speed_hz: u32) -> Result<()> {
+        let supports_variable_speed = self.get_firmware_version()? >= FirmwareVersion(1, 0, 0);
+        let speed_hz = speed_hz.to_le_bytes();
```

Comment:
Super nit: I have a slight preference for inlining `speed_hz.to_le_bytes()` on line 312.

Reply:
Nevermind, I guess that `to_le_bytes()` returns an array, not a slice.

Reply:
Yep, it returns an array while `sent_ctrl` expects a slice.